.github/workflows/release.yml

```
@@ -13,9 +13,15 @@ jobs:
         run: |
           export GITHUB_REF_NAME="${{ github.ref_name }}"
           export RELEASE_VERSION="${GITHUB_REF_NAME#v}"
+          export RELEASE_MINOR_VERSION="${RELEASE_VERSION%.*}"
+          export RELEASE_MAJOR_VERSION="${RELEASE_MINOR_VERSION%.*}"

```

To be prepared for other variants of the semver2 tags i would slightly modifiy the script to pull out major and minor:

```suggestion
          export RELEASE_MAJOR_VERSION="${RELEASE_VERSION%%.*}"
          export remaining_version="${RELEASE_VERSION#*.}"
          export RELEASE_MINOR_VERSION="${RELEASE_MAJOR_VERSION}.${remaining_version%%.*}"

```

---

For which kind of tag do we get a difference?
I'm not that familiar with the substitution syntax...